### DISCUSSION TITLE
On MacOs (ARM3Pro) running Sonoma 14.2.1 the RustDesk client UI is not responsive to input or mouse selection

### DISCUSSION THREAD

**Discussion by Bounder** [rustdesk/rustdesk#7165] [Q&A]
OS: Sonoma 14.2.1
Chip: Apple M3 Max 

When using the latest release MacOs(Arm) version of the RustDesk client, I can't operate the client because I cannot change anything in the settings screen. I can open the settings, but the window is not responding to user input. 

Is anyone else experiencing problems with RustDesk on Mac OS Arm? I transferred all settings/programs from my old Macbook which was Intel-based, could this cause problems with the RustDesk client UI? I did uninstall and installed different versions and they all have this problem.

**Comment by iexcept** [comment]
Have you tried to completely uninstall it? Search for any leftovers and remove them. Typically they store something in `rustdesk` dirs

**Comment by Bounder** [comment]
I searched for the common locations on MacOs but couldn't find any. Is there a list somewhere that tells the location of rust desk files or setting files?

**Comment by Bounder** [comment]
Found the answer. Deleting /Users/arie/Library/Preferences/com.carriez.RustDesk solved my problem.

**Comment by smeusekentik** [comment]
Oh, thank you for this. It was driving me crazy...
